ruff.toml

```diff
@@ -1,4 +1,4 @@
 line-length = 110
 
-[per-file-ignores]
+[lint.per-file-ignores]
```

Comment:
Seems like this change is causing the Github workflow to fail - any reason for this change?

Reply:
Yes - I needed to install ruff and the compile did not give any errors with using a newer version (0.3.2 vs 0.0.286).

Reply:
Makes sense - can you update this part (https://github.com/0xFableOrg/roll-op/blob/master/devenv.sh#L48) to use 0.3.2 instead of 0.0.286?